### PR TITLE
Upgrade to SpatialOS 13.0.0

### PR DESCRIPTION
**Pull request by ghost**
Following the guide from https://github.com/spatialos/UnitySDK/blob/master/docs/migration.md

Tested locally to build and run

Tested deployment and works as expected

Followed entry on Test Matrix and it passes

### CODE REVIEW

File: workers/unity/Assets/Plugins/Improbable/Sdk/Src/Editor/Editor/.iOSBuildTools/XcodeSettingsBuilder.cs [hunk 1]
```diff
@@ -0,0 +1,103 @@
+
+using System.IO;
+using UnityEditor;
+using UnityEditor.iOS.Xcode;
+using UnityEngine;
+
+namespace Improbable.Editor.Addons.iOSBuildTools
```
Comment (jessicafalk) [inline workers/unity/Assets/Plugins/Improbable/Sdk/Src/Editor/Editor/.iOSBuildTools/XcodeSettingsBuilder.cs:7]: completely didn't see that this still exists. @jared-improbable we either want to add either update that file in the repo or remove it. The current version does not work correctly, I believe.

Not really relevant for this PR though

**Review by jessicafalk — APPROVED** [review]
assuming that you just followed the migration steps and everything runs just fine on windows and mac.

**Comment by jessicafalk** [comment]
also: the migration guide mentions that the tutorials don't have the gitignore set up correctly. I think we can remove that warning after all tutorials have been updated ;)

**Review by jared-improbable — APPROVED** [review]

File: workers/unity/Assets/Plugins/Improbable/Sdk/Src/Editor/Editor/.iOSBuildTools/XcodeSettingsBuilder.cs (hunk 1 above)
Comment (oblm) [reply workers/unity/Assets/Plugins/Improbable/Sdk/Src/Editor/Editor/.iOSBuildTools/XcodeSettingsBuilder.cs:7]: I've confirmed that this is superfluous but harmless. I've raised an internal ticket to remove it in future versions.

**Comment by ghost** [comment]
Testing on Mac too before merging

**Review by oblm — APPROVED** [review]

**Comment by ghost** [comment]
Works on Mac as well ✔️

**Pull request merged**